cilantro_audit/widgets/auditor_page.kv

```diff
@@ -12,6 +14,7 @@
                 size_hint: None, None
                 pos_hint: {"center_x": -0.8, "center_y": 0.8}
                 on_release:
+                    root.manager.get_screen(AUDITOR_PAGE).get_audit_templates(root.manager)
```

Comment:
I believe you don't need to do that nor the import on top `#:import AUDITOR_PAGE...`.  Your branch needs to be merged with master since some classes are still working under an older version (i.e: `view_audit_templates`). This is why the list was not populated. The function call for populating the templates list at `view_audit_templates` constructor was commented out in the original file. This, however, is not the case in the most updated version of `view_audit_templates`.

So, I would suggest you merge whatever you have now with the current version of master to make sure everything is working correctly and will not break other working functionalities.